ledger/shelley/genesis.go

```diff
@@ -128,6 +204,32 @@ func (g *ShelleyGenesis) GenesisUtxos() ([]common.Utxo, error) {
 	return ret, nil
 }
 
+// GetInitialPools returns all initial stake pools with their delegators
+func (g *ShelleyGenesis) GetInitialPools() (map[string]GenesisPool, map[string][]string, error) {
```

Comment:
We should return native ledger types here like we do above for `GenesisUtxos()`. At minimum, we should use `common.Address` where appropriate.